pkg/service/node.go

```diff
@@ -61,11 +62,19 @@ func (n *NodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 	}
 
 	// is there a filesystem on this device?
-	filesystem, err := getDeviceInfo(device)
+	var filesystem string
+	for i := 0; i < 5; i++ {
```

Comment:
Instead of the fixed retries, could the number of retries be adjusted via config map? This setting should be adjustable even when the pod is running. Implement it in a future PR only if you think it is necessary.

Reply:
i can be a chart override and command line param.  I will create an issue.